nemo_text_processing/text_normalization/de/taggers/time.py

```diff
@@ -43,37 +43,35 @@ def __init__(self, deterministic: bool = True):
         labels_minute_single = [str(x) for x in range(1, 10)]
         labels_minute_double = [str(x) for x in range(10, 60)]
 
-        delete_leading_zero_to_double_digit = (pynutil.delete("0") | (NEMO_DIGIT - "0")) + NEMO_DIGIT
+        delete_leading_zero_to_double_digit = (pynutil.delete("0").ques | (NEMO_DIGIT - "0")) + NEMO_DIGIT
```

Comment:
`.ques` is unnecessary ( and potentially damaging here).

Reply:
Happy to remove it, but without it, comma-separated digit strings (e.g. `1,2,3,4`) won't normalize , causing the tests to fail.

Reply:
Without applying a closure over at most one occurrence time expressions in the following format `h.mm` will not normalize correctly.

Reply:
Oh this is for time, nvm. Confusing with cardinal.